backend/.env

```
@@ -44,9 +44,8 @@
 
 PORT=4000
 MONGO_URI=
-JWT_SECRET=someSecretKey
 # BASE_URL=https://7a1db250d5b7.ngrok-free.app
-BASE_URL=https://9289-150-129-158-206.ngrok-free.app
+BASE_URL=https://4911-106-192-120-98.ngrok-free.app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BASE_URL key should go before the MONGO_URI key

(UnorderedKey)